src/fluent/watch.ts

```diff
@@ -346,7 +438,7 @@ export class Watcher<T extends GenericClass> {
           this.#events.emit(WatchEvent.RECONNECT, this.#retryCount);
           this.#streamCleanup();
 
-          void this.#runner();
+          void this.#watch();
```

Comment:
```suggestion
          void this.#watch();
```

Do you think it makes sense to clear the RV here before re-establishing the URL incase we missed something?